### DISCUSSION TITLE
Rem support in v6

### DISCUSSION THREAD

**Discussion by 7iomka** [mantinedev/mantine#3375] [Q&A]
Hi.
I see https://github.com/mantinedev/mantine/commit/0ad20efebeb8d79da09467fa0ee7cbfdfc32ccd0
you added rem support in v6.
But the limitation in math operations with theme values is a huge breaking change for most users!
Why just to not use somewhere internally utility function called `rem` for transform all pixel values in rem?
For postcss I used `postcss-pxtorem`, and this is so nice, because we have best of both: 
- we keep writing values in pixels (This is really more familiar to most users than in the mind or in a calculator to translate the necessary values in rem or use the always function), we keep codebase practically unchanged.
- as a result we receive rem values which is the best practice in terms of support accessibility

**Comment by rtivital** [comment]
px are transformed to rem automatically when they are passed through props. It is not a good idea to continue using math operations with theme values – it is not clear whether emotion will be supported by the future versions of React. Using `calc` for math operations will ensure that values can be replaced with CSS variables.

**Reply by GerardKetuma** [reply]
"it is not clear whether emotion will be supported by the future versions of React."

@rtivital What are the plans of mantine about emotion not being supported in future React versions. Thanks again for all your time maintaining this library.

**Reply by rtivital** [reply]
If it is not supported then it will not be used
